app/api/swagger/swaggerconfig.js

```diff
@@ -11,7 +13,6 @@ export default (app) => {
       description: `Uwazi is an open-source solution for building and sharing document collections.
       <br>Remember that using the "Try it out" functionality will execute the requests over your local instalation!`
     },
-    host: 'localhost:3000',
```

Comment:
Not sure this is the wanted behavior.  Check other comment.

Reply:
The issue reported Swagger to fail when accessed on a production build using a FQDN. We should discuss if we want the "Try it out" functionality to work on a production deployment. Not removing the host implies that any Try it out action on a domain different than localhost won't work.
I'll double check it, this is still in progress.